src/test/java/com/google/step/snippet/external/W3SchoolsClientTest.java

```diff
@@ -43,6 +43,36 @@ public void jsonCodeCard() {
     assertEquals(expected, actual);
   }
 
+  @Test
+  public void cssEscaping() {
+    Card actual = client.search("https://www.w3schools.com/html/html_css.asp", "css style");
```

Comment:
I think the logic for determining when to do HTML escaping should also be checked. Could you add tests where you check:
1. Query does not contain escape filter, but webpage does. In this case, the HTML should still be escaped. 
Example: `client.search("https://www.w3schools.com/html/html_css.asp", "style");`
2. Query contains escape filter, but webpage does not. In this case, the HTML should still be escaped.
Example: `client.search("https://www.w3schools.com/js/js_errors.asp", "html js error");`
3. Neither query nor webpage contains escape filter. In this case, the HTML should not be escaped.
Example: `client.search("https://www.w3schools.com/js/js_errors.asp", "js error");`